themes/default/content/docs/concepts/projects/project-file.md

```diff
@@ -60,6 +60,7 @@ The runtime attribute has an additional property called `options` where you can
 | `buildTarget` | Only applies to the `go` runtime | Path to save the compiled go binary to. |
 | `binary` | applies to the `go`, `dotnet`, and `java` runtimes | Path to a pre-built executable. |
 | `virtualenv` | Only applies to the `python` runtime | Virtual environment path. |
+| `typeChecker` | Only applies to the `python` runtime | Type checker library to use. |
```

Comment:
Is it `typeChecker` or `typechecker`? The test in https://github.com/pulumi/pulumi/pull/15725/files#diff-b5d5cca2844a1618591e5ca532f5b69a1035243c876f46194e2aef2a1dbe6f3fR7 is all lowercase.

Reply:
Oh good catch, test is right but I had a time where I was deciding between cases so my memory had recalled the camelCase variety